pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -533,6 +534,8 @@ func manageTemplate(rawTemplate string, imagePullSpec string, operatorImagePullS
 		OperatorImage:               operatorImagePullSpec,
 		Verbosity:                   verbosity,
 		GracefulTerminationDuration: gracefulTerminationDuration,
+		// 35s for minimum-termination-duration, 5s extra cri-o's graceful termination period
+		SetupContainerTimeoutDuration: gracefulTerminationDuration + 35 + 5,
```

Comment:
this does not look like a duration.

Reply:
?

Reply:
you adding to duration :-) which I think makes the result duration too...

Reply:
bad naming. It should be GracefulTerminationSeconds. Probably the same with your change.